src/app/editor/editor.component.scss

```diff
@@ -104,4 +104,8 @@ div.right-side-container {
   height: 100vh;
   padding-left: $container-padding;
   border-left: 2px solid lightgray;
+}
+
+.flash {
+  border: 2px solid yellow;
 }
```

Comment:
It will be in another pr where I fix tslint errors (which also forces to add new-line at the end of `*.ts` files.)